test/onnx/test_op_consistency.py

```diff
@@ -0,0 +1,462 @@
+# Owner(s): ["module: onnx"]
+
+"""Test consistency between torch.onnx exported operators and torch operators.
+
+Usage:
+
+    pytest test/onnx/test_op_consistancy.py
+
+    To run tests on a specific operator (e.g. torch.ceil):
+
+    pytest test/onnx/test_op_consistancy.py -k ceil
+
+    Read more on Running and writing tests:
+        https://github.com/pytorch/pytorch/wiki/Running-and-writing-tests
+
+Note:
+
+    When new ops are supported, please scroll down to modify the EXPECTED_SKIPS_OR_FAILS and
+    ALLOWLIST_OP lists. See "Modify this section"
+
+"""
+
+import copy
+import dataclasses
+import io
+import unittest
+from typing import (
+    AbstractSet,
+    Callable,
+    Collection,
+    Iterable,
+    Optional,
+    Sequence,
+    Tuple,
+    Union,
+)
+
+import onnx
+
+import torch
+from torch.onnx import _constants, verification
+from torch.testing._internal import (
+    common_device_type,
+    common_methods_invocations,
+    common_utils,
+)
+from torch.testing._internal.opinfo import core as opinfo_core
+
+# The min onnx opset version to test for
+MIN_ONNX_OPSET_VERSION = 9
+# The max onnx opset version to test for
+MAX_ONNX_OPSET_VERSION = _constants.ONNX_MAX_OPSET
+
+TESTED_OPSETS = range(MIN_ONNX_OPSET_VERSION, MAX_ONNX_OPSET_VERSION + 1)
+ORT_PROVIDERS = ("CPUExecutionProvider",)
+
+SUPPORTED_DTYPES = (
```

Comment:
Minor nit because I was pulled in here: this tuple and the ones below seem to carry the same information. Why not create this one from the others like

```py
SUPPORTED_DTYPES = (
    *BOOL_TYPES,
    *INT_TYPES,
    ...
)
```

Reply:
Done